javascript/javascript1/week4/voiceassistant.js

```diff
@@ -0,0 +1,70 @@
+
+let result = "";
+let toDo = [];
+
+function getReply(command) {
+  let commandArray = command.split(" ");
+
+  if (command === "Hello my name is Benjamin") {
+    result = commandArray[commandArray.length - 1];
+    return `Your name is ${result}`;
```

Comment:
Try to modify this part, in order to voice assistant could remember any name, not only Benjamin.